src/chatlog/chatlog.cpp

```diff
@@ -204,8 +214,74 @@ void ChatLog::mouseMoveEvent(QMouseEvent* ev)
 
     QPointF scenePos = mapToScene(ev->pos());
 
+    int leftColumnWidth = Settings::getInstance().getColumnLeftWidth();
+    int rightColumnWidth = Settings::getInstance().getColumnRightWidth();
+    int endLocation = width() - rightColumnWidth - margins.left() - verticalScrollBar()->sizeHint().width() - 15 / 2;
```

Comment:
What are these magic numbers 15 and 2? Why do you divide 15/2 here? Can't just subtract another magic (int)7.5? Is 15 font dependent?